intellij-java-google-style.xml

```diff
@@ -2,14 +2,14 @@
 <code_scheme name="GoogleStyle">
     <option name="JAVA_INDENT_OPTIONS">
         <value>
-            <option name="INDENT_SIZE" value="2" />
-            <option name="CONTINUATION_INDENT_SIZE" value="4" />
-            <option name="TAB_SIZE" value="8" />
-            <option name="USE_TAB_CHARACTER" value="false" />
-            <option name="SMART_TABS" value="false" />
-            <option name="LABEL_INDENT_SIZE" value="0" />
-            <option name="LABEL_INDENT_ABSOLUTE" value="false" />
-            <option name="USE_RELATIVE_INDENTS" value="false" />
+          <option name="INDENT_SIZE" value="2" />
```

Comment:
The rest of this XML file seems to use 4 spaces for indentation instead of 2. Should clean up the diff nicely.